test/BUILD.gn

```diff
@@ -644,6 +644,7 @@ if (!is_android) {
       "//brave/chromium_src/third_party/blink/renderer/core/frame/reporting_observer_browsertest.cc",
       "//brave/chromium_src/third_party/blink/renderer/modules/battery/navigator_batterytest.cc",
       "//brave/chromium_src/third_party/blink/renderer/modules/bluetooth/navigator_bluetoothtest.cc",
+      "//brave/chromium_src/third_party/blink/renderer/modules/quota/navigator_storagetest.cc",
```

Comment:
We no longer want to add tests directly to //brave/test/BUILD.gn. They are supposed to go into BUILD.gn where the test lives and then added as a dep here. (A recent example is https://github.com/brave/brave-core/pull/6964).

Reply:
- [x] moved into `//chromium_src/third_party/blink/renderer/modules/BUILD.gn`